exec/src/interfaces.mli

```diff
@@ -3,7 +3,6 @@
  *
```

Comment:
This part of the patch is weird...

Reply:
I mean: git did a weird job in interpreting this patch as a renaming + contents change.

Reply:
Each patch taken individually does not suffer this issue, but indeed git heuristic are a bit lost here.

(This is why we should have used Pijul :D)